tests/attr/test_gradient_shap.py

```diff
@@ -173,7 +175,11 @@ def test_basic_relu_multi_input(self):
         attributions_ig = ig.attribute(inputs, baselines=baselines)
         self._assert_shap_ig_comparision(attributions, attributions_ig)
 
-    def _assert_shap_ig_comparision(self, attributions1, attributions2):
+    def _assert_shap_ig_comparision(
+        self,
+        attributions1: Union[Tensor, Tuple[Tensor, ...]],
```

Comment:
This method should only take Tuple[Tensor, ...] for both attributions1 and attributions2. Note that you may need to use cast (imported from typing) in places that call this function to clarify that you are passing a tuple of tensors.